src/System.Net.Security/src/System/Net/Unix/SSPISecureChannelType.cs

```diff
@@ -0,0 +1,290 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+using System.Globalization;
+using System.Net.Security;
+using System.Runtime.InteropServices;
+using System.Security.Authentication;
+using System.Security.Authentication.ExtendedProtection;
+using System.Security.Cryptography.X509Certificates;
+
+namespace System.Net
+{
+    // For SSL connections:
+    internal class SSPISecureChannelType : SSPIInterface
+    {
+        public Exception GetException(SecurityStatus status)
+        {
+            // TODO: To be implemented
+            throw new Exception("status = " + status);
```

Comment:
Why is this different from the original code anyway? AFAIK Unix code will still supposed to throw Win32Exception correctly setting the OS-level error code.